doc/decoding/decoding_tutorial.rst

```diff
@@ -466,15 +468,20 @@ Construct the new estimator object and use it in a pipeline::
 
 and recompute the cross-validation score::
 
-    >>> cv_scores = cross_val_score(anova_lda, X, y, cv=cv, verbose=1)  # doctest: +SKIP
+    >>> cv_scores = cross_val_score(anova_lda, fmri_masked, target, cv=cv, verbose=1)  # doctest: +SKIP
     >>> classification_accuracy = np.mean(cv_scores)  # doctest: +SKIP
+    >>> n_conditions = len(set(target))  # number of target classes
     >>> print("Classification accuracy: %.4f / Chance Level: %.4f" % \
-    ...    (classification_accuracy, 1. / n_conditions)) # doctest: +SKIP
-    Classification accuracy: 1.0000 / Chance level: 0.5000
+    ...    (classification_accuracy, 1. / n_conditions))  # doctest: +SKIP
+    Classification accuracy: 0.7846 / Chance level: 0.5000
```

Comment:
Hmm, the change is a bit frightening. I thinks that its is because the test that was discussed in that script actually refers to another script, namely `plot_haxby_anova_svm.py`.
So the change is right, never mind.